keps/sig-api-machinery/365-paginated-lists/README.MD

```diff
@@ -40,7 +122,18 @@ Relevant issues:
 
 * https://github.com/kubernetes/kubernetes/issues/2349
 
-## Terminology
+### Goals
```

Comment:
is it a goal to maintain the inconsistent pagination used by the storage version migrator?

Reply:
I'm afraid I'm missing some context here. Can you clariy or point me to the issue?